dali/base/dafdesc.cpp

```diff
@@ -3198,6 +3198,30 @@ void initializeStorageGroups(bool createPlanesFromGroups)
                 }
             }
         }
+
+        //Walk the drop zones, and add them as storage groups if they have no servers configured, or "."
+        Owned<IRemoteConnection> conn = querySDS().connect("/Environment/Software", myProcessSession(), 0, 2000);
+        if (conn)
+        {
+            Owned<IPropertyTreeIterator> dropzones = conn->queryRoot()->getElements("DropZone");
+            ForEach(*dropzones)
+            {
+                IPropertyTree & cur = dropzones->query();
+                unsigned numServers = cur.getCount("ServerList");
+                if (numServers <= 1)
+                {
+                    //If no instances then assume it is always local (e.g. azure)
+                    const char * ip = numServers ? cur.queryProp("ServerList[1]/@server") : nullptr;
+                    IPropertyTree * plane = storage->addPropTree("planes");
+                    plane->setProp("@name", cur.queryProp("@name"));
+                    plane->setProp("@prefix", cur.queryProp("@directory"));
+
+                    //Temporary.  This is likely to change to eventually create a hostgroup and set the @hosts property.
+                    if (ip && !streq(ip, "."))
+                        plane->setProp("@host", ip);
```

Comment:
This was a very temporary fix.  You don't want the hosts to be duplicated, so you would either need to follow the name hosts link to another section of information, or patch the tree up after reading from the config.  I opted for something "wrong" to avoid messing up implementing the correct thing later.

Reply:
fair enough